src/main/resources/config/init.xml

```diff
@@ -33,9 +33,8 @@
             <![CDATA[
                 var timeout = mc.getProperty("timeout");
                 if (timeout == null || timeout == "") {
-                    timeout = 30000;
+                    mc.setProperty("timeout", 30000);
```

Comment:
is this timeout property specific to the connector?

Reply:
Looks the timeout property is relates to the connector. In that case let's add some uncommon prefix. timeout is kind of very common property name which can be used by a user. and that may override users' property. 
Since this is common problem in connectors @abeykoon

Reply:
Yes we use it in the init to set the call timeout. So should we change it to something like "dayforceCallTimeout"?